### PR TITLE
Manual outlier_frame_extraction bugfix

### PR DESCRIPTION
**Pull request by n-poulsen**
Addresses issue #2279.

When manually extracting outlier frames from a video (not located in the `{project}/videos/` folder), two issues were occurring:

- the video was not added to the project
- the path to the video was given to `napari-deeplabcut`, instead of the path to alias (or copied video) created in the `{project}/videos/` folder.

There `napari-deeplabcut` tried to extract the frames to a `labeled-images` folder located in the parent folder of the analyzed video (which logically did not exist). This pull request fixes these issues.

### CODE REVIEW

File: deeplabcut/refine_training_dataset/outlier_frames.py [hunk 1]
```diff
@@ -619,6 +631,57 @@ def compute_deviations(
...
+    from deeplabcut.create_project import add
+
+    # make sure coords and videos are a list
+    videos = [video]
+    if coords is not None:
+        coords = [coords]
+
+    try:
+        add.add_new_videos(config, videos, coords=coords, copy_videos=copy_videos)
+    except ValueError as err:
```
Comment (jeylau) [inline deeplabcut/refine_training_dataset/outlier_frames.py]: What would the ValueError be caused by?

Comment (jeylau) [reply deeplabcut/refine_training_dataset/outlier_frames.py]: Improper `coords`?

**Comment by jeylau** [comment]
Should we pin tensorflow to `<2.13` to avoid failing tests?

File: deeplabcut/refine_training_dataset/outlier_frames.py (hunk 1 above)
Comment (n-poulsen) [reply deeplabcut/refine_training_dataset/outlier_frames.py]: this should just be `except` catching all exceptions (as in the existing code, this line was used for debugging). fixing now

**Comment by n-poulsen** [comment]
> Should we pin tensorflow to `<2.13` to avoid failing tests?

Should we do this in a separate pull request or directly here?

**Comment by jeylau** [comment]
> > Should we pin tensorflow to `<2.13` to avoid failing tests?
> 
> Should we do this in a separate pull request or directly here?

I'd do it separately, and merge it in `main` before this one.

**Review by jeylau — APPROVED** [review]

**Review by MMathisLab — APPROVED** [review]

**Pull request merged**